WordPress/Classes/ViewRelated/Me/All Domains/View Models/AllDomainsListViewModel.swift

```diff
@@ -10,7 +10,7 @@ class AllDomainsListViewModel {
     // MARK: - Properties
 
     @Published
-    private(set) var state: State = .empty
+    private(set) var state: State = .normal
```

Comment:
The initial `state` value is changed from `.empty` to `.normal` because `.empty` takes now a param.